__mocks__/fs.js

```diff
@@ -0,0 +1,80 @@
+"use strict";
+
+const path = require("path");
+const fs = jest.genMockFromModule("fs");
+
+/**
+ * In-memory store for our mock directories and their files
+ */
+let mockFiles = Object.create(null);
```

Comment:
There wasn't really a specific reason for it on my end, I was referencing an example from the docs and they used `Object.create(null)` instead of assigning mockFiles to an empty object. 

I can make the change in assigning mockFiles to an empty object here.